action.yml

```
@@ -43,4 +45,4 @@ runs:
         INPUT_ANACONDA_NIGHTLY_UPLOAD_TOKEN: ${{ inputs.anaconda_nightly_upload_token }}
         INPUT_ANACONDA_NIGHTLY_UPLOAD_LABELS: ${{ inputs.anaconda_nightly_upload_labels }}
       run: |
-        pixi run upload_wheels.sh
+        pixi run $GITHUB_ACTION_PATH/upload_wheels.sh
```

```suggestion
        pixi run ${{ github.action_path }}/upload_wheels.sh
```

While `${{ github.action_path }}` and `$GITHUB_ACTION_PATH` are the same given https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables I think it would be nicer to have the be the same text so it is easier to `git grep` for it in the future.